examples/00-basic/10-math_operations.py

```diff
@@ -0,0 +1,154 @@
+# noqa: D400
+"""
+.. _ref_math_operators_example:
+
+Mathematical Operations
+~~~~~~~~~~~~~~~~~~~~~~~
+
+DPF provides operators implementing mathematical operations,
+ranging from addition and multiplication to FFT and QR solving
+
+For a complete list, see :ref:`ref_dpf_operators_reference`, under the math section.
+
+"""
+
+# Import the necessary modules
+import ansys.dpf.core as dpf
+
+###############################################################################
+# Addition
+# ~~~~~~~~
+
+# Initialize Fields
+num_entities = 2
+field1 = dpf.Field(nentities=2)
+field2 = dpf.Field(nentities=2)
+
+# By default Fields contain 3d vectors
+# So with 3 entities we need 9 values
+field1.data = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]
+field2.data = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]
+
+field1.scoping.ids = range(num_entities)
+field2.scoping.ids = range(num_entities)
+###############################################################################
+# Once the fields are ready we can instantiate an operator
+add_op = dpf.operators.math.add(field1, field2)
+# Alternatively:
```

Comment:
@ansys-akarcher Could you remove the comments ? We don't want to use "dpf.Operator("add")" syntax in documentation.  Main reason is that "dpf.operators.math.add" is more pythonic and has intellisense associated